src/components/Security/Roles/List.vue

```diff
@@ -119,6 +119,15 @@ export default {
     )
   },
   methods: {
+    async deleteRoles(index, collection, ids) {
+      try {
+        await this.performDelete(index, collection, ids)
+        this.$set(this.selectedDocuments, this.selectedDocuments.splice(0, this.selectedDocuments.length))
+        this.fetchRoles()
+      } catch (e) {
+        return Promise.reject(e)
```

Comment:
(nitpicking) Throwing an error inside an async function is exactly like returning a rejected promise, so you can get rid of the whole `try..catch` statements.